### PR TITLE
Feature/buy button tests

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
Add tests to the buy-button component

#### What problem is this solving?
Is possible to verify that the button is working properly

#### How should this be manually tested?

yarn test

#### Screenshots or example usage

image

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: packages/buy-button/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.1.8] - 2018-26-04
+
+* **Added** prepublish to package.json and tests.
```
Comment (lucasecdb) [inline packages/buy-button/CHANGELOG.md]: follow the correct semantic like

```markdown
## [0.1.8] - 2018-04-26

### Added

* Points go here
```

File: packages/buy-button/src/__tests__/BuyButton.test.js [hunk 2]
```diff
@@ -1,16 +1,40 @@
...
+  const valExpected = 3
+
+  const props = {
+    quantity: 1,
+    skuId: 1,
+    seller: '1',
+    salesChannel: '1',
+    orderFormId: '12321',
+    mutate: jest.fn(),
+    afterClick: function() {
```
Comment (lucasecdb) [inline packages/buy-button/src/__tests__/BuyButton.test.js]: Use `jest.fn()` to check if the function is called

Comment (salesfelipe) [reply packages/buy-button/src/__tests__/BuyButton.test.js]: 👍

File: packages/buy-button/src/index.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ import addToCartMutation from './mutations/addToCartMutation.gql'
 /**
  * BuyButton Component. Adds a list of items to the cart.
  */
-class BuyButton extends React.Component {
+export class BuyButton extends React.Component {
```
[the remark this answers is not in the document]
Comment (salesfelipe) [reply packages/buy-button/src/index.js]: 👍

File: packages/buy-button/CHANGELOG.md [hunk 4]
```diff
@@ -7,19 +7,26 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+## [0.1.8] - 2018-05-03
+
+### Added
+
+* **Package Config** Prepublish to package.json.
+* **Tests** Component Tests
+
+## [0.1.5] - 2018-04-26
 
 ### Bugfix
```
[the remark this answers is not in the document]
Comment (salesfelipe) [reply packages/buy-button/CHANGELOG.md]: 👍

File: packages/buy-button/src/__tests__/BuyButton.test.js [hunk 5]
```diff
@@ -1,16 +1,36 @@
...
+  })
 
+  it('should be rendered', () => {
     expect(wrapper).toBeDefined()
   })
+
+  it('should call afterClick', () => {
+    expect(props.afterClick.mock.calls.length).toBe(0)
+    wrapper.find('Button').simulate('click')
+    expect(props.afterClick.mock.calls.length).toBe(1)
```
Comment (lucasecdb) [inline packages/buy-button/src/__tests__/BuyButton.test.js]: Should also test if the mutate function was called

Comment (salesfelipe) [reply packages/buy-button/src/__tests__/BuyButton.test.js]: 👍

**Pull request merged**